data/kea-dhcp/kea-dhcp4.conf

```diff
@@ -0,0 +1,461 @@
+// This is a basic configuration for the Kea DHCPv4 server. Subnet declarations
```

Comment:
No hard requirement, but I would crop the config file down to the required settings and refer to https://kea.readthedocs.io for more details. That gives us a better overview about the USED settings in the test run, as opposed to show all POSSIBLE settings.

But I guess that's more of a style question, so I leave it to you to decide if you like that or not.